.github/workflows/tft.yml

```diff
@@ -180,7 +180,7 @@ jobs:
           tf_scope: private
           api_key: ${{ secrets.TF_API_KEY_RH }}
           update_pull_request_status: false
-          tmt_hardware: '{ "memory": ">= ${{ needs.prepare_vars.outputs.memory }} MB" }'
+          tmt_hardware: '{ "memory": ">= ${{ needs.prepare_vars.outputs.memory }} MB", "network": [{"type": "eth", "device-name": "eth0"}, {"type": "eth", "device-name": "eth1"}] }'
```

Comment:
The commit message is a bit confusing, it is not adding two NICs , you only added 1 additional NIC, `eth0` is still the default device for bridging the ssh connection between the control node and the managed host. Can you change the commit message to  `ci: Configure two NICs for test purposes` ?